Gemfile

```diff
@@ -15,8 +15,8 @@ gem 'rails-api'
 gem 'figaro'
 gem 'config'
 gem 'pg'
-gem 'json-schema'
-gem 'active_model_serializers'
+gem 'json-schema', '2.7.0' # apivore keeps us at 2.8.0 at max, current version is 3.0.30!! -- 8/3/17
```

Comment:
apivore is keeping our json-schema version down to 2.8.0 but even that change breaks 3-4 specs. The latest version of json-schema is 3.0.30! Need to discuss value or importance of keeping this gem up to date when apivore dependency requires it so much lower.